src/o365/graph/commands/teams/teams-channel-add.ts

```diff
@@ -0,0 +1,149 @@
+import auth from '../../GraphAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import GraphCommand from "../../GraphCommand";
+import * as request from 'request-promise-native';
+import Utils from '../../../../Utils';
+import { Channel } from './Channel';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  groupId : string;
+  name: string;
+  description?: string;
+}
+
+class TeamsChannelAddCommand extends GraphCommand {
```

Comment:
Following the naming convention, the command should be named `GraphTeamsChannelAddCommand`